### ISSUE TITLE
Option to show spell rank

### ISSUE THREAD

**Issue by Willy130683** [Nevcairiel/Quartz#32]
Hi can you please add an option to show the spell rank in the bar. If there is an option for this already I would appreciate a hint where to find it ;)

**Comment by aloucks** [comment]
Yes, this would be extremely helpful when downranking healing spells. Displaying the spell rank used to be configurable in aan older version.

https://github.com/Nevcairiel/Quartz/blob/79e9ad887d2b038aef178ac319ec915469dd5070/CastBarTemplate.lua#L135
